config.ts

```diff
@@ -164,6 +164,7 @@ export const defaultConfig = {
     'grpc': path.join(__dirname, 'src/plugins/plugin-grpc.js'),
     'hapi': path.join(__dirname, 'src/plugins/plugin-hapi.js'),
     'http': path.join(__dirname, 'src/plugins/plugin-http.js'),
+    'https': path.join(__dirname, 'src/plugins/plugin-https.js'),
```

Comment:
If I require `https`, that will recursively require `http` hitting the existing plugin. Would that not work?

Reply:
That won't be the case -- the order of events happens like this:

1. call `require('https')`
1. load plugin for `https`
1. load `https`
    1. call `require('http')`
    1. load plugin for `http`
    1. load `http`
    1. patch `http` (and `https`)
1. patch `https` (no op)

We can't patch `https` in (3iv) because it's already been loaded.

So what I'm proposing here is for (3i) thru (3iv) to be moved under (2) rather than (3).